src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide(num) {
+      this.result /= num;
```

Comment:
Consider adding a check to handle division by zero in the `divide` method to prevent potential runtime errors or unexpected results.